openstates/mt/__init__.py

```diff
@@ -82,3 +82,8 @@ def get_organizations(self):
         yield legislature
         yield upper
         yield lower
+
+    def get_session_list(self):
+        from openstates.utils.lxmlize import url_xpath
```

Comment:
Not a particular reason. just copied the `function` from `mt.py` file and updated the import path.